src/frontend/components/administration/apikey/ApikeyTab.vue

```diff
@@ -1,34 +1,26 @@
 <template>
   <v-card>
     <v-toolbar color="secondary" dark flat dense>
-      <v-toolbar-title v-text="$t('administration.apikey.title')" />
+      <v-toolbar-title> {{ $t('administration.apikey.title') }} </v-toolbar-title>
       <v-spacer />
       <v-icon>mdi-security</v-icon>
     </v-toolbar>
     <v-card-actions>
       <v-spacer />
       <v-btn
         @click.stop="setVisible(true)"
-        v-text="$t('administration.apikey.buttonCreate')"
-      />
-
+      >
+        {{ $t('administration.apikey.buttonCreate') }}
+      </v-btn>
       <CreateDialog
         v-model="createDialogVisible"
         @created="getApikeys()"
         @closed="setVisible(false)"
       />
     </v-card-actions>
 
-    <v-row
-      v-if="apikeys.length === 0"
-      align="center"
-      justify="center"
-      class="ma-2"
-    >
-      <v-col class="text-center" cols="12" sm="4">
-        {{ $t("administration.apikey.noApikeys") }}
-      </v-col>
-    </v-row>
+    <Loader v-if="loading" />
+    <NoData v-else-if="apikeys.length === 0" :local-key="'administration.apikey.noApikeys'" />
```

Comment:
`apikeys` may be undefined

```suggestion
    <NoData v-else-if="!apikeys?.length" :local-key="'administration.apikey.noApikeys'" />
```